saleor/webhook/payloads.py

```diff
@@ -311,11 +311,19 @@ def generate_checkout_payload(checkout: "Checkout"):
         "discount_name",
         "private_metadata",
         "metadata",
+        "channel",
     )
     user_fields = ("email", "first_name", "last_name")
     shipping_method_fields = ("name", "type", "currency", "price_amount")
     lines_dict_data = serialize_checkout_lines(checkout)
 
+    # todo use the most appropriate warehouse
+    warehouse = None
```

Comment:
This should be changed when we merge the checkout reservation feature - https://github.com/mirumee/saleor/pull/7589